cmd/compare.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/open-feature/cli/internal/manifest"
+	"github.com/spf13/cobra"
+)
+
+func GetCompareCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "compare",
+		Short: "Compare two manifest files",
+		Long:  `Compare two manifest files and list the changes`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 2 {
+				return fmt.Errorf("please provide two manifest files to compare")
```

Comment:
In #93 , I took a different direction, and included argument names for `--source` and `--target`. 

I could update this PR to follow the same pattern before merge if we like.